third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -65,8 +65,6 @@ resource "google_composer_environment" "test" {
       service_account = google_service_account.test.name
     }
   }
-
-  depends_on = [google_project_iam_member.composer-worker]
```

Comment:
Why is the explicit dependency no longer required?

Reply:
That dependency isn't required. It takes time for the GKE cluster to be provisioned and for Airflow to start making API calls with that service account. The IAM grant will be finished and consistent long before the cluster is ready.